### PR TITLE
updates order of activities and links on module pages

### LINKED ISSUE #2405: We need to update /activities
We should update [/activities](https://learning.mozilla.org/activities) to better organize our core offerings and curated offerings, as well as to use our most up-to-date versions of each core module/lesson/curriculum template.

We should update /activities in 2 steps.

# Better organize /activities

1. We should create a "core content" header at the top of our activities list an include these modules under it:

- Offline icebreakers
- Web Literacy Basics I
- Web Literacy Basics II
- Make Your First Webpage
- Privacy Basics: Protect Your Data
- Intermediate Web Literacy I: Intro to CSS
- Intermediate Web Literacy II: Storytelling with Scripts

2. We should create a "curated content" header after the core content section and list all remaining modules under that.

This gives users a quick and dirty way to filter our core content (which may also feature Mozilla-specific features like links to our badging work) from curated or community-generated content until we have a database or search UI.

# Link to updated activities for core content
@flukeout and @chadsansing should make sure the core modules in [this repo](https://github.com/mozilla/curriculum-final) are good to go, then we should link to these modules and activities from /activities.

This will give us a single repo for Mozilla content - and with Mozilla access - in the future.

cc @anmechung, @hannahkane, @ldecoursy, @Pomax

### PR DESCRIPTION
**Pull request by hannahkane**
- [x] Localization addressed - NA

Fixes #2405

**Changes proposed in this pull request:**
- rearranges items on Activities page
- updates links in five of the module pages

### CODE REVIEW

**Comment by hannahkane** [comment]
@chadsansing - would you mind reviewing these changes?

**Comment by chadsansing** [comment]
@hannahkane - I can't easily read the comparison, but as long as the order of core activities modules is this...

-Offline icebreakers
- Web Literacy Basics I
- Web Literacy Basics II
- Make Your First Webpage
- Privacy Basics: Protect Your Data
- Intermediate Web Literacy I: Intro to CSS
- Intermediate Web Literacy II: Storytelling with Scripts

...I think we're good.

**Comment by hannahkane** [comment]
That is the order, @chadsansing. Thanks.

@flukeout - could you do a code review?

**Comment by flukeout** [comment]
Testing & reviewing shortly.

File: pages/intermediate-web-lit-two.jsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ var curriculumList = [
         caption: "CC0 Public Domain",
         subtitle: "Communication, Creativity, Compose, Code, Remix",
         description: "Learn how HTML, CSS, and JavaScript connect to one another by using buttons and scripts to tell a simple story.",
-        link: "http://chadsansing.github.io/curriculum-testing/intermediate-web-lit-two/session01-buttons-and-alerts.html"
+        link: "https://mozilla.github.io/curriculum-final/intermediate-web-lit-two/session01-buttons-and-alerts.html#overview"
```
Comment (flukeout) [inline pages/intermediate-web-lit-two.jsx]: Not a big deal, but you don't have to add the ``#overview`` to the link.

**Comment by flukeout** [comment]
So yeah looks good, I have a suggestion for improving the styling of the ``h2``. If you crack open ``less/pages/activities.less`` and add the following to the top of the page...

```
.page.teaching-materials h2 {
  margin: 1.75em 0 1em 0;
}
```

...the spacing will look better. Other than that, good to go!

**Comment by chadsansing** [comment]
Huzzahs!

**Comment by hannahkane** [comment]
@flukeout - did I do it right?

**Comment by chadsansing** [comment]
Any notion of when this goes to production?

**Comment by hannahkane** [comment]
@chadsansing - I will see if someone is available to push to prod today.

**Comment by hannahkane** [comment]
@chadsansing - this is live now

**Comment by chadsansing** [comment]
Thank you, @hannahkane & team #updateactivities!

**Comment by chadsansing** [comment]
@hannahkane - any idea what happened to the line on each activity right before the web literacy skills in the overview that said, "Follow this link to remix this activity on Thimble!"?

**Comment by chadsansing** [comment]
@hannahkane - nvm; fixed!

**Pull request merged**